lib/DDG/Spice/BikeSharing/CitiBikeNYC.pm

```diff
@@ -10,6 +10,12 @@ spice wrap_jsonp_callback => 1;
 spice is_cached => 0;
 spice proxy_cache_valid => '200 304 15m';
 
+spice alt_to => {
+	status => {
```

Comment:
mhhh... that doesn't work. Remember that this is initially "namespaced" by the directory structure, so the right URL should be `/js/spice/bike_sharing/citi_bike_nyc/status`.

Removing `citi_bike_nyc` obviously doesn't work, because we are one dir below that. But the inner URL rewriting system doesn't seem to understand nested files, returning the same for both `/js/spice/bike_sharing/citi_bike_nyc/status` and (the non-existent) `/js/spice/bike_sharing/citi_bike_nyc/fuuuuu`; in both cases, it returns the first level reponse